courses/guidance/05.md

```diff
@@ -0,0 +1,35 @@
+---
+index: 5
+title: 5. Components Of a Vulcan Project
+author: Sumaa Sayed
+headerDepth: 3
+---
+
+## 5. Components
+
+### 5.1 What is a Component?
+[add text here]
+
+### 5.2 Creating a Component for Your Project
```

Comment:
Maybe it is suited for another section, but might want to list different ways to create a new component ( copy from spreadsheet, copy component, duplicate) and the purpose for each.